docs/Gemfile.lock

```diff
@@ -0,0 +1,205 @@
+GEM
```

Comment:
If this file is to be `.gitignore`d (as it should be as I understand it), then this file shouldn't also be checked into the repository.